flair/data.py

```diff
@@ -368,6 +420,44 @@ def convert_tag_scheme(self, tag_type: str = 'ner', target_scheme: str = 'iob'):
         for index, tag in enumerate(tags):
             self.tokens[index].add_tag(tag_type, tag)
 
+    def _infer_space_after(self):
+        """
+        Heuristics in case you wish to infer whitespace_after values for tokenized text. This is useful for some old NLP
+        tasks (such as CoNLL-03 and CoNLL-2000) that provide only tokenized data with no info of original whitespacing.
+        :return:
+        """
+        last_token = None
+        quote_count: int = 0
+        # infer whitespace after field
+
+        for token in self.tokens:
+            if token.text == '"':
+                quote_count += 1
+                if quote_count % 2 != 0:
+                    token.whitespace_after = False
+                elif last_token is not None:
+                    last_token.whitespace_after = False
+
+            if last_token is not None:
+
+                if token.text in ['.', ':', ',', ';', ')', 'n\'t', '!', '?']:
+                    last_token.whitespace_after = False
+
+                if token.text.startswith('\''):
+                    last_token.whitespace_after = False
+
+            if token.text in ['(']:
+                token.whitespace_after = False
+
+            last_token = token
+        return self
```

Comment:
The return value can be removed as it is not used (also from the docs).
Also the method name should not start with `_` as it is used outside the sentence class.